cli/CHANGELOG.md

```diff
@@ -202,6 +202,9 @@ _Released 5/21/2024_
 
 **Dependency Updates:**
 
+- Upgraded `electron` from to `27.3.10` to `28.3.1`.
+- Upgraded bundled Node.js version from `18.17.1` to `18.18.2`.
+- Upgraded bundled Chromium version from `118.0.5993.117` to `120.0.6099.56`.
```

Comment:
We're running the 29 upgrade first. Electron 28 contains a breaking change, so we're likely to skip this and it will be dead. We'll revisit if we have to though. https://github.com/cypress-io/cypress/pull/30224